### ISSUE TITLE
SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.

### ISSUE THREAD

**Issue by xwang1024** [babel/babel#9854]
## Bug Report

**Current Behavior**
An error occurred when I use **@babel/polyfill**  in the iOS 8 Safari.

```
SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.
```

The code that caused the error is in:
`node_modules/_@babel_polyfill@7.4.3@@babel/polyfill/lib/index.js`
```js
"use strict";

require("./noConflict");

function _global() {
  const data = _interopRequireDefault(require("core-js/library/fn/global")); // this line

  _global = function () {
    return data;
  };

  return data;
}
```

**Input Code**
src/index.js
```js
import "@babel/polyfill"
```

**Babel Configuration (.babelrc)**

```json
{
  "presets": [
    ["env", {
      "targets": {
        "browsers": ["iOS >= 6.0", "Android >= 4.0"],
      }
    }]
  ]
}
```

**Environment**
```
    "@babel/core": "^7.4.3",
    "@babel/polyfill": "^7.4.3",
    "babel-loader": "^8.0.5",
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "webpack": "^3.12.0",
    "webpack-cli": "^3.3.0",
    "webpack-dev-server": "^2.11.5"
```
- Node version: v8.11.3
- OS: macOS 10.14.4
- How you are using Babel: as a loader

**Additional context/Screenshots**

![image](https://user-images.githubusercontent.com/12004140/56078826-f0977b80-5e1e-11e9-9b71-bca2fbd87a5f.png)

**Comment by nicolo-ribaudo** [comment]
Fixed by https://github.com/babel/babel/pull/9812

**Comment by Jasper0819X** [comment]
I have encountered this problem too. Is there any workaround on it?

**Comment by patotoma** [comment]
I used a workaround to manually change this one `const` into `var` in the build files before deploying.

**Comment by nicolo-ribaudo** [comment]
Fixed in v7.4.4
